tests/e2e/etcd_config_test.go

```diff
@@ -274,6 +276,117 @@ func TestEtcdPeerNameAuth(t *testing.T) {
 	}
 }
 
+// TestEtcdPeerLocalAddr checks that the inter peer auth works with when
+// the member LocalAddr is set.
+func TestEtcdPeerLocalAddr(t *testing.T) {
+	e2e.SkipInShortMode(t)
+
+	nodeIP, err := netutil.GetDefaultHost()
+	fmt.Println("Got node IP:", nodeIP)
```

Comment:
`netutil.GetDefaultHost` isn't supported on non-Linux platform. Although other platforms (e.g. Darwin) are only tier 3 support, it isn't convenient if we can't run an e2e test locally on Darwin.

Suggest to get a non-loopback address with an alternative solution. 

FYI. https://gosamples.dev/local-ip-address/